apps/web/src/lib/data/window/preset-storage.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * @license BSD-3-Clause
+ * Copyright (c) 2023, ッツ Reader Authors
+ * All rights reserved.
+ */
+
+import { Subject, BehaviorSubject } from 'rxjs';
+import { localStorage } from './local-storage';
+
+export class PresetStorage {
+  private _preset: string;
+
+  private bookPresets$: BehaviorSubject<Record<string, string>>; // idk how better to avoid cyclic dependency
```

Comment:
moving the global book presets and presetstorage from store to outside of the class in this file should work (probably requires a ide restart so the linter stops complaining)